Packs/ForcepointSecurityManagementCenter/Integrations/ForcepointSecurityManagementCenter/README.md

```diff
@@ -0,0 +1,971 @@
+Forcepoint SMC provides unified, centralized management of all models of Forcepoint engines whether physical, virtual or cloud—across large, geographically distributed enterprise environments.
+This integration was integrated and tested with version xx of Forcepoint Security Management Center
```

Comment:
I see it's possible to set an API version in the Session object - https://fp-ngfw-smc-python.readthedocs.io/en/latest/pages/reference.html#smc.api.session.Session.api_version
guess if it's not provided, it takes the latest one. it's a good idea to check if we want to specify a constant version, to ensure that nothing will break in future versions  - but need to check if it's possible to define the same version for all customers.
Regardless, replace `xx` with the API version you were using to develop the integration with.

Reply:
I agree, we need to provider the version to make sure that nothing will break in future versions

Reply:
Added it